src/sc_crawler/vendors/_vultr.py

```
@@ -326,6 +391,36 @@ def _get_plans_metal():
     return response.json()["plans_metal"]
 
 
+def _vultr_auth_headers() -> dict[str, str]:
+    try:
+        api_key = environ["VULTR_API_KEY"]
+    except KeyError:
+        raise KeyError("Missing environment variable: VULTR_API_KEY")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)